client/src/services/SpecialAddresses.js

```diff
@@ -23,8 +23,8 @@ export default {
        min_amount: 10
      },
      {
-       name: 'WMP_1',
```

Comment:
It would be nice not to forget the first address, but put WMP_1 on status down, and add a new one WMP with status up